doc/source/install.rst

```diff
@@ -122,6 +122,10 @@ options you want to change and configure:
 |                     | if you use the Redis storage driver.              |
 +---------------------+---------------------------------------------------+
 
+The same options are also available as `incoming.<drivername>_` for configuring
```

Comment:
I meant _*, as whatver starts with drivername-underscore. Maybe I should add a * to make it more clear?